### PR TITLE
Add documentation for pulling docker images

### PR DESCRIPTION
**Pull request by DnlLrssn**
**Checklist**

- [ ] tests added
- [ ] commits conform to the [Commit message format](./CONTRIBUTING.md#commit)
- [x] documentation updated

**Squash commit message**

Describe how to pull docker image from Github packages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,3 +85,41 @@ Do the following:
    * `go get github.com/some/lib@v1.2.3`: Get a particular version.
 2. Run `go mod tidy` to remove any unused modules.
 3. Run `go mod verify` to add packages needed for test packages etc.
+
+## Pulling the docker image
```
Comment (carlcervin) [inline README.md]: Change to "Docker" throughout the text (where applicable)?

File: README.md [hunk 2]
```diff
@@ -85,3 +85,41 @@ Do the following:
...
+
+Unfortunately the Github packages docker repo is not very "public", more on this can be found in this community [thread](https://github.community/t5/GitHub-Actions/docker-pull-from-public-GitHub-Package-Registry-fail-with-quot/td-p/32782/page/4). This means a docker login needs to be done before the images can be pulled. To do this follow these steps:
```
Comment (carlcervin) [inline README.md]: Change to "GitHub" throughout the text (where applicable)?
Change ..."a docker login needs to be done"... to ..."a Docker login is needed before..."?

File: README.md [hunk 3]
```diff
@@ -85,3 +85,41 @@ Do the following:
...
+
+1. Create a new token with the scope `read:packages` [here](https://github.com/settings/tokens).
+2. Save your token to e.g. to a file (or use environment variable or similar).
```
Comment (carlcervin) [inline README.md]: Change ..."or use environment variable"... to ..."or use an environment variable"...?

File: README.md [hunk 4]
```diff
@@ -85,3 +85,41 @@ Do the following:
...
+3. Login with docker to `docker.pkg.github.com`.
+
+Using token saved to the file github.token:
```
Comment (carlcervin) [inline README.md]: Change ..."Using token saved to the"... to ..."Using a token stored in the"...?

File: README.md [hunk 5]
```diff
@@ -85,3 +85,41 @@ Do the following:
...
+2. Save your token to e.g. to a file (or use environment variable or similar).
+3. Login with docker to `docker.pkg.github.com`.
+
+Using token saved to the file github.token: 
+
+```bash
+docker login -u yourgithubusername --password=$(cat github.token) docker.pkg.github.com
+```
+
+Using token in environmental variable GITHUB_TOKEN:
```
Comment (carlcervin) [inline README.md]: Change ... "Using token in environmental variable"... to "Using a token in an environmental variable"...?

File: README.md [hunk 6]
```diff
@@ -85,3 +85,41 @@ Do the following:
...
+
+```bash
+docker login -u yourgithubusername --password=$GITHUB_TOKEN docker.pkg.github.com
+```
+
+4. Pull the the docker image.
```
Comment (carlcervin) [inline README.md]: Remove the extra "the".

File: README.md [hunk 7]
```diff
@@ -85,3 +85,41 @@ Do the following:
...
+
+Specific released version:
+
+```bash
+docker pull docker.pkg.github.com/qlik-oss/gopherciser/gopherciser:0.4.10
+```
+
+### Using the image in Kubernetes
+
+To use the image in Kubernetes, e.g. to perform executions as part of a Kubernetes job, credentials the Github package registry needs to be added the same way a private registry is used, see documentation [here](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/).
```
Comment (carlcervin) [inline README.md]: "creadentials the Github package" - is a word missing in there?
Change ..."registry needs"... to ..."registry need"...?

File: README.md (hunk 5 above)
Comment (DnlLrssn) [reply README.md]: "Using a token in the environmental variable GITHUB_TOKEN" better since its this specific one?

Comment (carlcervin) [reply README.md]: Ah, sorry, maybe better with "Using the token in the environmental..."?

File: README.md (hunk 1 above)
Comment (DnlLrssn) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (DnlLrssn) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (DnlLrssn) [reply README.md]: done

**Pull request merged**